rs/config/src/embedders.rs

```diff
@@ -238,6 +241,9 @@ pub struct Config {
 
     /// The maximum size of the stable memory.
     pub max_stable_memory_size: NumBytes,
+
+    /// The dirty page overhead factor for Wasm64.
+    pub wasm64_dirty_page_overhead_multiplier: u64,
```

Comment:
Nit: Maybe move the definition closer to the existing `dirty_page_overhead` parameter?

Reply:
done;